### PR TITLE
Add justification document for tf_42679

### PR DESCRIPTION
**Pull request by fridex**
Related: https://github.com/thoth-station/adviser/pull/1211 https://github.com/tensorflow/tensorflow/issues/42679

### CODE REVIEW

File: _j/tf_42679.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: TensorFlow in version 2.3 can produce misleading output summary
+---
+
+TensorFlow in version 2.3 can produce misleading output summary when
+initialized in a specific way.
+
```
Comment (goern) [inline _j/tf_42679.md:7]: do we have an an example of that misleading output?

**Review by goern — CHANGES_REQUESTED** [review]

File: _j/tf_42679.md (hunk 1 above)
Comment (fridex) [reply _j/tf_42679.md:7]: In the linked issue.

Comment (goern) [reply _j/tf_42679.md:7]: yes yes, do we want to show it on out page too? #SEO

Comment (fridex) [reply _j/tf_42679.md:7]: Added in 7cb024b

**Review by goern — APPROVED** [review]

**Pull request merged**